nexus/src/app/background/driver.rs

```diff
@@ -2,10 +2,13 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
-//! Manages execution of background tasks
+//! Generic system for managing execution of background tasks
+//!
+//! None of this file is specific to Nexus or any of the specific background
+//! tasks in Nexus, although the design is pretty bespoke for what Nexus needs.
```

Comment:
Yeah, I was thinking we may want to separate it into a different package in the workspace (`nexus/background-tasks` or something), not because I think it's a very general abstraction but just on the general principle of separating easily separable things to avoid a big honkin' Nexus crate.

Reply:
Yeah, I think it would be nice to do that at some point. Hopefully it would benefit compile times a bit...